repository/src/main/java/org/apache/atlas/repository/graph/GraphBackedSearchIndexer.java

```diff
@@ -130,7 +130,7 @@ public GraphBackedSearchIndexer(AtlasTypeRegistry typeRegistry) throws AtlasExce
         this.typeRegistry = typeRegistry;
 
         //make sure solr index follows graph backed index listener
-        addIndexListener(new SolrIndexHelper(typeRegistry));
+        //addIndexListener(new SolrIndexHelper(typeRegistry));
```

Comment:
Why is this commented and not removed?

Reply:
We can remove it directly, but just letting it there to have traceability

Reply:
Let got take care of traceability and version control. Git history stays preserved, so tracking it via that would still be possible if we remove it.

Reply:
I think we can remove it.

Reply:
Removed it